### ISSUE TITLE
In Mac App body type form data is not detecting.[bug]: 

### ISSUE THREAD

**Issue by shibilpk** [hoppscotch/hoppscotch#3734]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When i give text body using form-data it is not taken by request. It always shows that the field is required.

### Steps to reproduce

1. Take Post api
2. Give the values as form data.
3. Sent the request.
<img width="1440" alt="Screenshot 2024-01-09 at 2 15 56 PM" src="https://github.com/hoppscotch/hoppscotch/assets/59109749/629688f0-c9a7-4d08-85d8-3ec69f3baee1">

### Environment

Release

### Version

Cloud

**Comment by shibilpk** [comment]
Yes. it is only on my desktop application. Work fine in the app. Problem
exists in mac application.

**Comment by shibilpk** [comment]
Yes. it is only on my desktop application. Work fine in the web app.
Problem exists in mac application.

**Comment by retrokit-max** [comment]
I have experienced the exact same behaviour on the desktop app but on Windows 11. 

If I set the request body type to multipart/form, no body variables are making it through with the request. I copied the curl code for the same request, ran it manually from a terminal, and the same request executed successfully.

**Comment by krwadhwani** [comment]
I'm having the same issue on Mac Apple Silicon desktop app.

**Comment by liyasthomas** [comment]
Thanks for your patience and valuable feedback. We’ve fixed this bug in the latest release.
[Download the latest builds](https://hoppscotch.com/download) and provide your valuable feedback.

Closing this ticket as this bug is now fixed in the recent version. Please feel free to reach out if you have any other concerns.
